src/EnergyPlus/DataHeatBalance.cc

```diff
@@ -97,6 +99,49 @@ using DataBSDFWindow::BSDFWindowInputStruct;
 
 // Functions
 
+Real64 SpaceData::sumHATsurf(EnergyPlusData &state)
```

Comment:
If we are going to keep functions going in here, sometime it might be nice to just rename this as HeatBalance.cc and .hh.